v22.2/cdc-transformations.md

```diff
@@ -151,14 +158,20 @@ Therefore, the first changefeed created:
 
 {% include_cached copy-clipboard.html %}
 ~~~sql 
-CREATE CHANGEFEED INTO 'scheme://sink-URI' WITH schema_change_policy='stop' AS SELECT * FROM movr.vehicle_location_histories WHERE left(ride_id::string, 1) IN ('0','1','2','3');
+CREATE CHANGEFEED INTO 'scheme://sink-URI-1' 
+WITH schema_change_policy='stop' 
+AS SELECT * FROM movr.vehicle_location_histories 
+WHERE left(ride_id::string, 1) IN ('0','1','2','3');
```

Comment:
This looks great, and copys&pastes!! Yes!! Sorry for leading you astray with `\` characters 9 months ago!! (I think something changed in our tooling, I could have sworn that I remember multi-line SQL without `\` characters failed in the SQL shell ~9mo ago, but who knows. In any case, this looks great!!